src/main/java/com/bootcamp/demo/restapi/ScooterController.java

```diff
@@ -40,10 +40,10 @@ public void postScooter(@RequestBody final Scooter scooter) {
         scooterService.createScooter(scooter);
     }
 
-    @PostMapping("/available")
+    @PostMapping("/available/{searchRadius}")
```

Comment:
Why use POST instead of GET? You don't change anything on the server and its not sensitive information (searchRadius + location).

Reply:
Because I need RequestBody for location and HTTP's GET method does not include a request body as part of the spec.